main/errors/index.ts

```diff
@@ -0,0 +1,84 @@
+import { init as initSentry, captureException, IPCMode } from '@sentry/electron'
+import log from 'electron-log'
+import type { Event } from '@sentry/types'
+
+import showUnhandledExceptionDialog from '../windows/dialog/unhandledException'
```

Comment:
this is a bit nitty but do you think it makes more sense to have this handler return some value and then have `main/index.js` read this value and launch a window accordingly? I feel it would be a little cleaner to have all window/view launching in one place rather than dialogs being created in other modules.

maybe have the handler return a boolean such as `handled` or `recovered` and then main can show the dialog based on that value? would also make the function more pure and easier to test

Reply:
👍

Reply:
I don't think there's enough code here to make returning a value worthwhile - just going to expose the sentry method.